helm/kube-prometheus-stack/values.yaml

```diff
@@ -362,6 +362,9 @@ kube-prometheus-stack:
   prometheusOperator:
     enabled: true
     admissionWebhooks:
+      deployment:
```

Comment:
That's for admission webhook though right?

Reply:
Not the actual operator?

Reply:
<img src="https://media4.giphy.com/media/62tdliJy2Kz0AtSTPC/giphy.gif"/>